src/sentry/feedback/serializers.py

```diff
@@ -0,0 +1,29 @@
+from typing import Any, Optional, TypedDict
```

Comment:
we can remove this as part of this PR and move it to the one for the feedback details endpoint.